### PR TITLE
Add Go 1.19 Support

### PR DESCRIPTION
**Pull request by Aoang**

### CODE REVIEW

**Comment by Aoang** [comment]
TBR

File: doc.go [hunk 1]
```diff
@@ -3,35 +3,53 @@ Package fasthttp provides fast HTTP server and client API.
...
+
+  - Per-request logger.
+
+  - Unique request id.
+
+  - Request start time.
+
+  - Connection start time.
+
+  - Request sequence number for the current connection.
```
Comment (erikdubbelboer) [inline doc.go:47]: These need to be sub bullet points like the original.

File: doc.go [hunk 2]
```diff
@@ -3,35 +3,53 @@ Package fasthttp provides fast HTTP server and client API.
...
+
+  - Maximum request header size.
+
+  - Maximum request body size.
+
+  - Maximum request execution time.
+
+  - Maximum keep-alive connection lifetime.
+
+  - Early filtering out non-GET requests.
```
Comment (erikdubbelboer) [inline doc.go:33]: These need to be sub bullet points like the original.

**Review by erikdubbelboer — CHANGES_REQUESTED** [review]
Thanks! Super nice pull request, just 2 minor fixes needed.

**Comment by Aoang** [comment]
@erikdubbelboer Go doc comments do not support nested lists. https://tip.golang.org/doc/comment#mistakes

Maybe we can mix list markers like the example? Since bullet markers do not introduce list items in a numbered list.

**Comment by erikdubbelboer** [comment]
Thanks!

File: .github/workflows/lint.yml [hunk 3]
```diff
@@ -9,14 +9,40 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: actions/setup-go@v2
+    - uses: actions/setup-go@v3
       with:
-        go-version: 1.18.x
+        go-version: 1.19.x
+
+    - name: Get Go cache paths
```
Comment (codefromthecrypt) [inline .github/workflows/lint.yml:16]: @Aoang curious why you ended up doing this? In wazero, we're noticing the cache doesn't seem to work for golangci-lint and wondering if there's some difference between this and hard-coding (considering it is only run with ubuntu). Any thoughts appreciated!

https://github.com/tetratelabs/wazero/blob/main/.github/workflows/commit.yaml#L46-L52

Comment (Aoang) [reply .github/workflows/lint.yml:16]: continue https://github.com/tetratelabs/wazero/pull/804

**Pull request merged**